index.html

```diff
@@ -0,0 +1,126 @@
+            <div><p class="orange hellos">"Hello! Sharing world"</p></div>
+            <h1 class="orange">CREATIVE COMMONS GLOBAL SUMMIT</h1>
+            <p class="orange hello">"Hello! Sharing world"</p>
+        </div>
+        <p class="black intro">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea</p>
+        <h2 class="black date">2015.10.15(THUR) ~ 16(FRI)</h2>
+        <p class="black location">@National Museum of Korea, Art Center Nabi and more</p>
+    </section>
+    <!--THE MAIN PROGRAM SECTION-->
+    <section class="second-section">
+        <h2 class="white">Main Program</h2>
+        <div class="second-section-container">
+            <div class="prog-feature">
+                <img src="./assets/icon-1.png" alt="Lecturer">
+                <h4 class="orange">Lecture</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-2.png" alt="CC Exhibition">
+                <h4 class="orange">Exhibition</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-3.png" alt="Forum">
+                <h4 class="orange">Forum</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-4.png" alt="Workshop">
+                <h4 class="orange">Workshop</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-5.png" alt="CC Ignite">
+                <h4 class="orange">CC Ignite</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+        </div>
+        <a href="#" class="white logo-link-1">Join CC Global Summit 2015</a>
+        <a href="#" class=" logo-link-2 white">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <!--THE FEATURED SPEAKERS SECTION-->
+    <section class="third-section">
+        <h2>Featured Speakers</h2>
+        <aside id="my-popup"></aside>
+    </section>
+    <footer>
+        <div class="partners">
+            <p class="grey partners-heading">Partners</p>
+            <ul class="partners-list">
+                <li class="grey">mozilla</li>
+                <li class="grey">Google</li>
+                <li class="grey">NAVER</li>
+                <li class="grey">daumkakao</li>
+                <li class="grey">airbnb</li>
```

Comment:
- [ ] The font style is not correct for this section. Kindly try to correct this issue.

Yours
![image](https://user-images.githubusercontent.com/53019916/127884395-200622d2-6216-4bde-83bc-1ca51b642cab.png)

Original 
![image](https://user-images.githubusercontent.com/53019916/127884490-c249dadd-b364-49ea-8a5b-6ea37da34ca5.png)